### PR TITLE
add license.

### PR DESCRIPTION
**Pull request by dereuromark**
Resolves https://github.com/FriendsOfCake/awesome-cakephp/issues/113

Using the one others like https://github.com/ziadoz/awesome-php do it.

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+ Copyright (C) 2004 Sam Hocevar <sam@hocevar.net>
+
+ Everyone is permitted to copy and distribute verbatim or modified
+ copies of this license document, and changing it is allowed as long
+ as the name is changed.
+
+            DO WHAT THE FUCK YOU WANT TO PUBLIC LICENSE
+   TERMS AND CONDITIONS FOR COPYING, DISTRIBUTION AND MODIFICATION
+
+  0. You just DO WHAT THE FUCK YOU WANT TO.
```
Comment (arthurvr) [inline LICENSE.md]: `0.`?

**Comment by arthurvr** [comment]
I would rather have liked public domain for lgtm.

**Comment by bravo-kernel** [comment]
This more than works for me, thanks!

**Comment by dereuromark** [comment]
;) for me too

**Pull request merged**